Resources/views/Admin/create.html.twig

```diff
@@ -0,0 +1,47 @@
+<div class="modal-header">
```

Comment:
If this form is to create an account, I would definitely call it "createAccount", not just "create".